cypress/e2e/dashboard.cy.ts

```diff
@@ -65,4 +73,45 @@ describe('Dashboard', () => {
     cy.contains('42.00').should('not.exist')
     cy.contains('12.00')
   })
+
+  it('links to the envelope filtered transactions', function () {
```

Comment:
```suggestion
  it('links to the envelope filtered transactions', () => {
```

Reply:
This needs to be a `function ()` for the later `this.budget` to work.

Reply:
Right, I remember. And I'm the one who told you this 🤦🏼‍♂️ Maybe we should add a comment? :D